lib/ui/shared/smart_widgets/text_parser/text_parser_viewmodel.dart

```diff
@@ -1,61 +1,62 @@
 import 'package:flutter_parsed_text/flutter_parsed_text.dart';
 import 'package:stacked/stacked.dart';
-import 'package:stacked_services/stacked_services.dart';
 
-import '../../../../app/app.locator.dart';
-import '../../../../utilities/enums.dart';
+import 'package:url_launcher/url_launcher.dart';
```

Comment:
This shouldn't be here.
Use the one in lib/utilities/utilities.dart
You can fix it up there if it's not working